lib/state/ui/reducer.ts

```diff
@@ -162,8 +178,8 @@ const showNavigation: A.Reducer<boolean> = (state = false, action) => {
     case 'SELECT_TRASH':
     case 'SHOW_ALL_NOTES':
       return false;
-    case 'App.showDialog':
-      if (action.dialog && action.dialog.type === 'Settings') {
+    case 'SHOW_DIALOG':
+      if (action.dialog && action.dialog === 'SETTINGS') {
```

Comment:
we can remove these safety checks now because TypeScript will warn if we reference a property that might be missing. in fact, it autocompletes for me purely based on the action type

![autoCompleteDialog mov](https://user-images.githubusercontent.com/5431237/77973762-16c16000-72aa-11ea-852b-52149a3f0072.gif)